app/Actions/CreateAndAttachRating.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Actions;
+
+use App\Models\Entry;
+use App\Models\Rating;
+use Illuminate\Support\Facades\Auth;
+use Lorisleiva\Actions\Concerns\AsAction;
+
+class CreateAndAttachRating
+{
+    use AsAction;
+
+    /**
+     * Expect an array of rating_data, create Rating and modify categories as required
+     *
+     * @param Entry $entry
+     * @param array $ratingData
+     */
+    public function handle(Entry $entry, array $ratingData)
+    {
+        $rating = Rating::create([
```

Comment:
to save the extra DB query we can just new an instance of it
```suggestion
        $rating = new Rating([
```